### ISSUE TITLE
Sanitizing error messages

### ISSUE THREAD

**Issue by renejuan** [rstudio/shiny-server#245]

**Comment by bborgesr** [comment]
So sorry we didn't make that clearer! @jcheng5, @trestletech, I can add a sentence about this wherever you think is most appropriate...

**Comment by renejuan** [comment]
No worries. I can think of a few things where this should be added.

1. I think the sanitized message should be changed to  `An error has occurred. Check your logs, contact the app author for clarification. The shiny server is sanitizing errors. For additional information see http://shiny.rstudio.com/articles/sanitize-errors.html`

Barbara; I would also modify you article to let people know that the shiny server default can be changed by adding `sanitize_errors`

2. I would also modify the default config file to have the setting `sanitize_errors true;`. Or because it is redundant we can add `sanitize_errors false;` but comment it out and add a little explanation

3. Finally; I would add some to the Shiny server configuration guide (http://docs.rstudio.com/shiny-server/)

**Comment by wayneworkman** [comment]
Yes, the config file should have all options present in it - even if they are commented out. And should have a comment per configuration line, just like default apache configurations or samba configurations.
